aten/src/ATen/native/mkldnn/MKLDNNCommon.cpp

```diff
@@ -117,6 +117,10 @@ ideep::tensor itensor_from_tensor(const Tensor& tensor) {
   }
 }
 
+int set_verbose(int level) {
+    return ideep::utils::set_verbose(level);
```

Comment:
I wonder, which ideep version introduced this call?

Reply:
It was added to ideep at https://github.com/intel/ideep/commit/e48e61e60a97e8f0c91ffc848744f692f0919df2.